includes/class-wc-siftscience-events.php

```diff
@@ -199,7 +199,7 @@ public function add_script() {
 			}
 
 			$name = 'wc-siftsci';
-			$path = plugins_url( 'dist/wc-siftsci.js', dirname( __FILE__ ) );
+			$path = plugin_dir_url( __DIR__ ) . 'dist/wc-siftsci.js';
```

Comment:
I agree with the change to __DIR__ (or __FILE__), but I think you should keep it as:

```
plugins_url( 'dist/wc-siftsci.js', __DIR__/__FILE__  );
```

Reply:
No need for this level of complicity we just nedd the plugin url 
Elaborate your point if view